client/api/src/client.rs

```diff
@@ -74,7 +75,7 @@ pub trait BlockchainEvents<Block: BlockT> {
 }
 
 /// Fetch block body by ID.
-pub trait BlockBody<Block: BlockT> {
+pub trait BlockProvider<Block: BlockT> {
```

Comment:
Could this be eventually moved to `primitives`? Like is there any reason why this is not part of `sp_blockchain`?

Reply:
Seems that we already have `HeaderBackend`, maybe this one should be called `BlockBackend` instead?